src/libs/actions/Session.js

```diff
@@ -112,7 +133,82 @@ function signOut() {
         .catch(error => Onyx.merge(ONYXKEYS.SESSION, {error: error.message}));
 }
 
+/**
+ * Checks the API to see if an account exists for the given login
+ *
+ * @param {String} login
+ */
+function hasAccount(login) {
```

Comment:
Maybe we could name this `fetchAccount()` instead? `hasAccount` sounds more like a utility method that would return `true` if `account.accountExists`.